collector/collector.go

```diff
@@ -64,40 +67,73 @@ type AttunityCollector struct {
 	// The key is server name.
 	ExcludedTasks []*TasksFilter
 
+	// A list of tags to get task details on
+	IncludedTags []*string
+
 	// How long to wait for API responses; 0=infinite
 	Timeout int
 
 	httpClient *http.Client
+
+	auth string
 }
 
 // NewAttunityCollector returns a pointer to an attunityCollector object
 // which implements the Prometheus Collector interface.
 // It should be registered to a Prometheus Registry.
-func NewAttunityCollector(cfg *Config) *AttunityCollector {
+func NewAttunityCollector(cfg *Config, ignoreCert bool) *AttunityCollector {
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: ignoreCert}
```

Comment:
This line still needs removed and the additional `ignoreCert` arg to the function definition isn't needed anymore.